Calculate.Tests/ProgramTests.cs

```diff
@@ -0,0 +1,42 @@
+using Xunit;
+using System;
+using System.IO;
+
```

Comment:
If you use the IntelliTect tool
```suggestion
using IntelliTect.TestTools.Console;
```